src/Model/Item.php

```diff
@@ -2769,7 +2769,7 @@ private static function putInCache(&$item)
 	 * @hook  prepare_body ('item'=>item array, 'html'=>body string, 'is_preview'=>boolean, 'filter_reasons'=>string array) after first bbcode to html
 	 * @hook  prepare_body_final ('item'=>item array, 'html'=>body string) after attach icons and blockquote special case handling (spoiler, author)
 	 */
-	public static function prepareBody(array &$item, $attach = false, $is_preview = false, $only_cache = false)
+	public static function prepareBody(array &$item, bool $attach = false, bool $is_preview = false, bool $only_cache = false): string
```

Comment:
You still need to make sure all the `return` calls of this method are returning a string as type-hinted.

Reply:
Yes, you can (and you should change it). It doesn't make sense for a string processing method to return void.

Reply:
Now I see it on my server, too: `PHP Fatal error:  A function with return type must return a value in /var/www/.../src/Model/Item.php on line 2814`.

Reply:
Okay, I changed it.